api/agent/protocol/json.go

```diff
@@ -40,6 +40,7 @@ type jsonIn struct {
 	Body        string          `json:"body"`
 	ContentType string          `json:"content_type"`
 	Protocol    CallRequestHTTP `json:"protocol"`
+	Version     string          `json:"version"`
```

Comment:
`version` is too vague imo. when we have function versioning this is too easy to conflate for users. `json_protocol_version` or something is less challenging to misinterpret

Reply:
yeah, in terms of protocol `version` is the "ok" name, but for request body let that be `protocol_version`